dist/manifest.json

```diff
@@ -33,7 +33,10 @@
       "https://superuser.com/*",
       "https://askubuntu.com/*",
       "http://stackoverflow.com/*",
-      "https://stackoverflow.com/*"
+      "https://stackoverflow.com/*",
+      "http://package.elm-lang.org/*",
+      "https://docs.rs/*",
+      "https://doc.rust-lang.org/*"
```

Comment:
This one I noticed the same problem you mentioned, we need some custom CSS in order to merge it.

URL tested: https://doc.rust-lang.org/std/ascii/trait.AsciiExt.html
Preview: http://cl.ly/kEm1